code/game/objects/structures/signs.dm

```diff
@@ -436,3 +431,68 @@
 		to_chat(user,"This one belongs to Dr.[claimant], MD.")
 	else
 		to_chat(user,"The name is left blank for some reason.")
+
+/obj/structure/sign/janitor
+	name = "\improper JANITORIAL CLOSET"
+	desc = "A sign indicating a room used to store cleaning supplies."
+	icon_state = "janitor"
+
+/obj/structure/sign/engineering
+	name = "\improper ENGINEERING"
+	desc = "A sign labelling an area as the Engineering department."
+	icon_state = "engineering"
+
+/obj/structure/sign/telecomms
+	name = "\improper TELECOMMUNICATIONS"
+	desc = "A sign labelling an area as the Telecommunications room."
+	icon_state = "tcomm"
+
+/obj/structure/sign/cargo
+	name = "\improper CARGO BAY"
+	desc = "A sign labelling the area as a cargo bay."
+	icon_state = "cargo"
+
+/obj/structure/sign/bridge
+	name = "\improper BRIDGE"
+	desc = "A sign indicating the Bridge. Not the kind you cross rivers with, the other kind."
+	icon_state = "bridge"
+
+/obj/structure/sign/forensics
+	name = "\improper FORENSICS"
+	desc = "A sign labelled FORENSICS."
+	icon_state = "forensics"
+
+/obj/structure/sign/security
+	name = "\improper SECURITY"
+	desc = "A sign labelling the area as belonging to Security."
+	icon_state = "sec_scale"
+
+/obj/structure/sign/security/alt
+	name = "\improper SECURITY"
```

Comment:
These don't need to be set again as they are inherited from the previous definition.

Reply:
These? Do you just mean security/alt name or am I missing another?

Reply:
Name and desc on this one, I didn't see any others.

Reply:
Oh just realized mobile view is really bad and didn't make it clear what line you were pointing at.